recipe/build_base.bat

```diff
@@ -8,10 +8,8 @@ ECHO [packages] >> mplsetup.cfg
 ECHO tests = False >> mplsetup.cfg
 ECHO sample_data = False >> mplsetup.cfg
 ECHO toolkits_tests = False >> mplsetup.cfg
-ECHO [libs] >> mplsetup.cfg
-ECHO system_freetype = True >> mplsetup.cfg
 
 set MPLSETUPCFG=mplsetup.cfg
```

Comment:
This environment variable, and the corresponding file can be removed.

Reply:
Dropped in commit: https://github.com/conda-forge/matplotlib-feedstock/pull/373/commits/8c8807577b1ec9784120069921b097eea35b6a36